mealie/core/security/jwt_validation.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+import requests
+from jwcrypto import jwk, jwt
+
+from mealie.core.config import get_app_settings
+
+
+def _get_jwks() -> jwk.JWKSet:
+    # JWKS is retrieved from the jwks endpoint
+    settings = get_app_settings()
+    jwk_data = requests.get(settings.JWT_AUTH_JWK_SET_URL)
+    return jwk.JWKSet().from_json(jwk_data.text)
```

Comment:
`settings.JWT_AUTH_JWK_SET_URL` will contain the URL in which the JWKS json is exposed, using Python `requests` this is retrieved in order to decode the JWT.